### DISCUSSION TITLE
Does DPLR support simulation of different temperatures in one box?

### DISCUSSION THREAD

**Discussion by Ding-Hairui** [deepmodeling/deepmd-kit#2195] [Q&A]
Hello, DPers.
I am planning to simulate the solid-liquid boundary of Fe with external electric field. According to my knowledge, the '[fix efield](https://docs.lammps.org/fix_efield.html)' and '[compute temp/region](https://docs.lammps.org/compute_temp_region.html)' commands in LAMMPS are related to this situation, and DPLR is more suitable for electric field. My questions are:
1. I've heard that some commands of LAMMPS are designed for specific potentials. Does DPLR support them?
2. If available, should the training data include solid-liquid boundary? Or dost DP just need data of solid and liquid structures?
Thanks for your reply!

**Comment by wanghan-iapcm** [comment]
You may modify the `fix dplr` command as 
```
fix		0 all dplr model ener.pb type_associate 1 3 bond_type 1 efield XX YY ZZ
```
To apply an external field of (XX, YY, ZZ) to the DPLR system. More details on DPLR is referred to [the doc](https://docs.deepmodeling.com/projects/deepmd/en/master/model/dplr.html#molecular-dynamics-simulation-with-dplr)

The training data should sample all relevant configurations in applications.
